src/components/app/App.tsx

```diff
@@ -2,7 +2,7 @@ import { AppBar, IconButton, Menu, MenuItem, Toolbar, Typography } from '@materi
 import * as MenuIcon from '@material-ui/icons/Menu';
 import * as React from 'react';
 import { HashRouter, NavLink, Route } from 'react-router-dom';
-import { ClientConfig } from '../../models/client_config';
+import { ClientConfig, IsValidConfig } from '../../models/client_config';
```

Comment:
is relative pathing really the only way to do imports?

Reply:
I think so to avoid ambiguity with imports of node_modules